apps/web/src/pages/JobPosterTemplateAdminPages/UpdateJobPosterTemplatePage/components/KeyTasksSection/Display.tsx

```diff
@@ -0,0 +1,53 @@
+import { useIntl } from "react-intl";
+
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { FragmentType, getFragment } from "@gc-digital-talent/graphql";
+import { htmlToRichTextJSON, RichTextRenderer } from "@gc-digital-talent/forms";
+
+import { InitialData_Fragment } from "./KeyTasksSection";
+
+interface DisplayProps {
+  initialDataQuery: FragmentType<typeof InitialData_Fragment>;
+}
+
+const Display = ({ initialDataQuery }: DisplayProps) => {
+  const intl = useIntl();
+  const notProvided = intl.formatMessage(commonMessages.notProvided);
+
+  const { tasks } = getFragment(InitialData_Fragment, initialDataQuery);
+
+  return (
+    <div className="grid gap-6 sm:grid-cols-2">
+      <div className="flex flex-col gap-6">
+        <h3 className="font-bold">
+          {intl.formatMessage({
+            defaultMessage: "English key tasks",
+            id: "FxP6EJ",
+            description: "Title for the English key tasks section",
+          })}
+        </h3>
```

Comment:
another instance of `Heading`.

Also @mnigh and @NienkeBr just did some work to normalize our field labels for French nad English. We may want to roll that into this PR.

Reply:
[switch to heading component](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14241/commits/23c7a9c226497621ddf1489727f9e7a1a7191e22)